bin/recog_match

```diff
@@ -33,6 +33,10 @@ option_parser = OptionParser.new do |opts|
     options.color = true
   end
 
+  opts.on("--multi-match", "Enable multiple matches") do
```

Comment:
I'd suggest:

```
opts.on("--[no-]multi-match", "Enable or disable multiple matches (defaults to disabled)") do |o|
  options.multi_match = o
end
```

This tells the user what the default is and gives them the ability to be explicit in its operation.